api-reference/endpoint/get-software.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "Get Software Info"
+api: "POST https://eol.xeol.com/api/v1/query"
+description: "This endpoint gets a software's support information"
+---
+
+### Body
+
+<ParamField body="identifier" type="string[]" required="true">
+  List of software identifiers.
+  
+  Can be either in `cpe`, `purl`, or `custom` format.
```

Comment:
Assume that i'm a user. I have just purchased extended support from RHEL version 8. I need to hit the Xeol API to figure out the support IDs associated with RHEL version 8.

How would I craft or find the CPE associated with RedHat in order to make this query?

Reply:
I think we should be matching on entity name here rather than identifiers. 

Additionally, this endpoint is inconsistent with /check where we have an object[] with `type` (cpe/purl/etc) and then the `identifier`

Reply:
`custom` allows you to put the software name that we can search on